components/Buttons/MarkPlayedButton.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-btn v-if="canMarkWatched" icon dark @click.stop.prevent="togglePlayed">
+    <v-icon :color="isPlayed ? 'blue' : 'white'"> mdi-check </v-icon>
```

Comment:
I *think* you could use the `primary` color in that case, does it work?

Reply:
Good idea! Updated